scripts/dfx_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+killall -9 node || true
+killall -9 icx-proxy || true
+dfx stop
+
+start_icx() {
+    killall icx-proxy
+    sleep 2
+    # Start ICX Proxy
+    dfx_local_port=$(dfx info replica-port)
+    icx-proxy --fetch-root-key --address 127.0.0.1:8545 --dns-alias 127.0.0.1:bd3sg-teaaa-aaaaa-qaaba-cai --replica http://localhost:$dfx_local_port &
+    sleep 2
+}
+
+cargo build --tests -p integration-tests --features dfx_tests
+
+rm -f dfx_tests.log
+
+set +e
+dfx start --background --clean 2> dfx_tests.log
```

Comment:
do not create temporary files in the root folder. Creates them inside `target` instead